lib/import/utility/extraction_utilities.rb

```diff
@@ -0,0 +1,152 @@
+require 'csv'
+# Common location for useful functions and constants
+module Import
+  module Utility
+    # Extraction utilities code
+    module ExtractionUtilities
+      # Common utility methods
+      class CommonUtility
+        # TODO: this pile of regexs is a mess and needs to be cleaned up
+        GENE_ONLY = '(?<location>c\.[^ \.]+)'.freeze
+        GENE_LOC = "#{GENE_ONLY} ?(?<protein>\(p\.[^\)\(]*\))?".freeze
+        BRCA_BASE = 'br?c?a?'.freeze
+        BRCA     = BRCA_BASE + '(?<brca>1|2)'.freeze
+        EXON_LOC = 'exons? (?<exons>\d+[a-z0-9]*(?:-\d+[a-z0-9]*)?)'.freeze
+        VARIANT  = '(?:familial)?(?<variantclass>(?: likely) pathogenic)?' \
+        ' BRCA(?<brca>1|2) (?:mutation|sequence variant)'.freeze
+        ZYGOSITY = '(?<zygosity>[a-z]+)zygous'.freeze
+        GENE_LOCATION = '(?<location>c\.[^ \.]+) ?(?<protein>\(p\.[^)]*\))?'.freeze
+        EXON_LOCATION = 'exons? (?<exons>\d+[a-z0-9]*(?:-\d+[a-z0-9]*)?)'.freeze
+        LOCATION_REGEX = /(#{GENE_LOCATION}(?: in (?:brca(?:1|2) )?#{EXON_LOCATION})?|#{EXON_LOCATION})/i
```

Comment:
@shilpigoeldev could you look at using the `/x` modifier to split the regexp over multiple lines? [Example here](https://stackoverflow.com/questions/3762183/split-ruby-regex-over-multiple-lines).

There are additional lines in this file that should also be corrected